api/well-known/README.md

```diff
@@ -0,0 +1,72 @@
+# Discovery APIs
+The APIs described here allow a user of the Veraison to obtain and view meta information about the Veraison deployment. Information about deployment can be queried according to the service e.g. provisioning or verification.
+
+The relevant resource is created in response to a client `GET`. The queried information is then outputted as shown in the examples below.
+
+## Well-known API: Provisioning service
+The information for the provisioning service has the following attributes:
+
+* The allowed provisioning media types;
+* The version of the provisioning service.
+
+### Querying information about the provisioning service
+
+- Request:
+```http
+GET /.well-known/veraison/provisioning
+Host: veraison.example
+```
+
+- Response:
+```http
+HTTP/1.1 200 OK
+Content-format: application/vnd.veraison.well-known-info+json
+
+{
+  "media-types": [
+    "application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1",
+    "application/corim-unsigned+cbor; profile=http://enacttrust.com/veraison/1.0.0",
+    "application/corim-unsigned+cbor; profile=http://arm.com/cca/ssd/1"
+  ],
+  "version": "commit-cb11fa0"
+}
+```
+
+
+## Well-known API: Verification service
+The information for the verification service has the following attributes:
+
+* The public key used to sign the evidence;
```

Comment:
Is it the public key to sign the evidence or it is the public part of the key to sign the Attestation Results, by a Verifier?

Reply:
s/evidence/attestation result/

Reply:
Is the public key used to sign both evidence and the attestation result (AR)?

Reply:
It is the Public part of the key used to Sign the Attestation Result in a Verifier.

The Public part of Evidence signing key should not be exposed in an API

Reply:
Ok so it should instead say "The public key used to sign the Attestation Result"

Reply:
I'd say "The public key to verify the Attestation Result"